frontend/public/components/storage/shared.ts

```diff
@@ -29,30 +30,33 @@ export const provisionerAccessModeMapping = {
   'pd.csi.storage.gke.io': ['ReadWriteOnce'],
 };
 export const initialAccessModes = ['ReadWriteOnce', 'ReadWriteMany', 'ReadOnlyMany'];
-export const accessModeRadios = [
+
+export const accessModeRadios: any = () => [
   {
     value: 'ReadWriteOnce',
-    title: 'Single User (RWO)',
+    title: i18next.t('public~Single user (RWO)'),
   },
   {
     value: 'ReadWriteMany',
-    title: 'Shared Access (RWX)',
+    title: i18next.t('public~Shared access (RWX)'),
   },
   {
     value: 'ReadOnlyMany',
-    title: 'Read Only (ROX)',
+    title: i18next.t('public~Read only (ROX)'),
   },
 ];
-export const volumeModeRadios = [
+
+export const volumeModeRadios: any = () => [
```

Comment:
Use a better return type than `any`. If you leave the return type off, TypeScript would infer something better here.